pkg/backup/backuper.go

```diff
@@ -0,0 +1,326 @@
+package snapshot
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	"github.com/mattn/go-sqlite3"
+
+	"github.com/pkg/errors"
+)
+
+// BackupFilenamePrefix is the prefix used in every backup file.
+const BackupFilenamePrefix = "tbl_backup"
```

Comment:
This is used as a prefix in every backup file. [See](https://github.com/tablelandnetwork/go-tableland/pull/227/files#diff-c95ef053206f27ba2307970f3b1bfaa55fcd806df5e69e44123c010ccf796ac7R231).